plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -33,13 +36,33 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
 		vendorName: '',
 		vendorUrl: '',
 		icon: '',
+		label: null,
+		primary_color: null,
 		url: '',
 		price: 0,
 		image: '',
 		averageRating: null,
 		reviewsCount: null,
 	};
 
+	function isSponsored(): boolean {
+		return SPONSORED_PRODUCT_LABEL === product.label;
+	}
+
+	/**
+	 * Sponsored products with a primary_color set have that color applied as a dynamically-colored stripe at the top of the card.
+	 * In an ideal world this could be set in a data- attribute and we'd use CSS calc() and attr() to get it, but
+	 * attr() doesn't have very good support yet, so we need to apply some inline CSS to stripe sponsored results.
+	 */
+	function inlineCss(): Object {
+		if ( ! isSponsored() || ! product.primary_color ) {
+			return {};
+		}
+		return {
+			background: `linear-gradient(${product.primary_color} 0, ${product.primary_color} ${SPONSORED_PRODUCT_STRIPE_SIZE}, white ${SPONSORED_PRODUCT_STRIPE_SIZE}, white)`
```

Comment:
This line gives a lint.

```suggestion
			background: `linear-gradient(${ product.primary_color } 0, ${ product.primary_color } ${ SPONSORED_PRODUCT_STRIPE_SIZE }, white ${ SPONSORED_PRODUCT_STRIPE_SIZE }, white)`,

```